components/l10n/common/localization_util.cc

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/l10n/common/localization_util.h"
+
+#include "base/strings/utf_string_conversions.h"
+#include "ui/base/resource/resource_bundle.h"
+
+namespace brave_l10n {
+
+std::u16string GetLocalizedResourceUTF16String(const int resource_id) {
```

Comment:
seems very odd to me that we need a helper for this, but upstream rarely calls `LoadLocalizedResourceString`. I see that this is used extensively so we can leave it for now, but I'd like to discuss for follow-up

Reply:
Agreed. I will raise a follow up issue to investigate.